src/kmedoids.jl

```diff
@@ -13,7 +13,7 @@ The output of [`kmedoids`](@ref) function.
  - `assignments::Vector{Int}`: the indices of clusters the points are assigned
    to, so that `medoids[assignments[i]]` is the index of the medoid for the
    ``i``-th point
- - `acosts::Vector{T}`: assignment costs, i.e. `acosts[i]` is the cost of
+ - `costs::Vector{T}`: assignment costs, i.e. `acosts[i]` is the cost of
```

Comment:
```suggestion
 - `costs::Vector{T}`: assignment costs, i.e. `costs[i]` is the cost of
```
Could you check that all references of `acosts` are fixed? E.g. in docs?